x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/summary_panel/datasets_quality_indicators.tsx

```diff
@@ -33,13 +33,24 @@ import { mapPercentagesToQualityCounts } from '../../quality_indicator';
 
 export function DatasetsQualityIndicators() {
   const { onPageReady } = usePerformanceContext();
-  const { datasetsQuality, isDatasetsQualityLoading, datasetsActivity } = useSummaryPanelContext();
+  const {
+    datasetsQuality,
+    isDatasetsQualityLoading,
+    datasetsActivity,
+    numberOfDatasets,
+    numberOfDocuments,
+  } = useSummaryPanelContext();
   const qualityCounts = mapPercentagesToQualityCounts(datasetsQuality.percentages);
   const datasetsWithoutIgnoredField =
     datasetsActivity.total > 0 ? datasetsActivity.total - datasetsQuality.percentages.length : 0;
 
-  if (!isDatasetsQualityLoading) {
-    onPageReady();
+  if (!isDatasetsQualityLoading && (numberOfDatasets || numberOfDocuments)) {
+    onPageReady({
+      key1: 'datasets',
```

Comment:
suggestion:

Since the custom metrics are pretty flexible and allows the consumer to pass any value  as number `(number, number representing ms, number representing seconds)` it would help for the analysis to the have more descriptive key names. 

ex
`datasets` -> `total_number_of_datasets` (or anything that could help understand what the value number holds. 

that's really up to you though :D